rq/worker.py

```diff
@@ -960,6 +982,10 @@ def _get_safe_exception_string(exc_strings):
                 exc_strings = [exc.decode("latin-1") for exc in exc_strings]
         return ''.join(exc_strings)
 
+    def push_job_success_handler(self,handle_func):
```

Comment:
I don't think we need this method, you can directly call `self._job_success_handlers.append()`